Formula/handbrake.rb

```diff
@@ -0,0 +1,26 @@
+class Handbrake < Formula
+  desc "Open-source video transcoder available for Linux, Mac, and Windows."
+  homepage "https://handbrake.fr/"
+  url "https://github.com/HandBrake/HandBrake.git", :revision => "d6e2e09ee7c00d9ed371d0a9b16ca87b6ee417e1"
+  version "1.0.0-alpha1"
```

Comment:
HandBrake 1.0.0 will be released in a matter of days. I believe the intent is to update the formula to the new release before merging.